app/jobs/connect_app_job.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class ConnectAppJob < ApplicationJob
+  def perform(app, token)
+    url = "https://n8n.openfoodnetwork.org.uk/webhook/regen/connect-enterprise"
+    event = "connect-app"
+    payload = {
+      enterprise_id: app.enterprise_id,
+      access_token: token,
+    }
+
+    response = WebhookDeliveryJob.perform_now(url, event, payload)
```

Comment:
Yay, nice to see the `WebhookDeliveryJob` in use!

Reply:
I just noticed this is a wrapper for `WebhookDeliveryJob`. Would it make sense to subclass it I wonder? I can't think it would make much difference..

Reply:
In doubt, I tend to avoid inheritance. This job could be seen as a special webhook job which also captures the response. But it's also doing UX stuff and it may expand one day. Only part of it is the webhook functionality. So I think that composition is the better pattern here.